etc/examples/paths.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# these can be set by folks running the examples/Makefile.testing to use a different binaries than in the path
+ECHO=${ECHO:-echo}
+JQ=${JQ:-jq}
+KUBECTL=${KUBECTL:-kubectl}
+PACHCTL=${PACHCTL:-pachctl}
+RM=${RM:-rm}
```

Comment:
AFAICS, this isn't being used

Reply:
rm is used by "make clean"

Reply:
That's called by the redundantly specified `RM` in the makefile

Reply:
It's not redundant; it's specified in there so it can be passed down to any scripts that are invoked, such as these.  Once it's specified, it's in the env.  The paths.sh will only set if not specified by the env, and the Makefile is asserting control over them so that the invoker of the script can control them.